java/src/test/java/org/rocksdb/BackupableDBOptionsTest.java

```diff
@@ -16,7 +17,7 @@
 
 public class BackupableDBOptionsTest {
 
-  private final static String ARBITRARY_PATH = "/tmp";
+  private final static String ARBITRARY_PATH = Environment.isWindows() ? System.getenv("TEMP") : "/tmp";
```

Comment:
I did not want to change behavior for other platforms so I fixed only Windows. Yes I can change it.